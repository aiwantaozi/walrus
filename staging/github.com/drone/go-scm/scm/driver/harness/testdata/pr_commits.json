[
  {
    "author": {
      "identity": {
        "email": "thomas.honey@harness.io",
        "name": "thomas.honey"
      },
      "when": "2023-02-09T17:12:10.976Z"
    },
    "committer": {
      "identity": {
        "email": "noreply@harness.io",
        "name": "Harness"
      },
      "when": "2023-02-09T17:12:10.976Z"
    },
    "message": "Create bla_file",
    "sha": "0c221fd126b9457d0ad2037641416083549f59c5",
    "title": "string"
  }
]