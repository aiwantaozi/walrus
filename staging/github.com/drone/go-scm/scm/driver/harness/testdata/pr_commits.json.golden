[
    {
        "Sha": "0c221fd126b9457d0ad2037641416083549f59c5",
        "Message": "Create bla_file",
        "Author": {
            "Name": "thomas.honey",
            "Email": "thomas.honey@harness.io",
            "Login": "",
            "Avatar": ""
        },
        "Committer": {
            "Name": "Harness",
            "Email": "noreply@harness.io",
            "Login": "",
            "Avatar": ""
        },
        "Link": ""
    }
]